---
layout: post
title: "Buenrostro, J. D. et al. Single-cell chromatin accessibility reveals principles of regulatory variation. Nature (2015). doi:10.1038/nature14590"
date: 2015-08-12
categories: ATAC-seq single-cell
---

[Link to paper](http://www.ncbi.nlm.nih.gov/pubmed/26083756)

An 'obvious' extension to their [earlier work]({% post_url 2015-07-28-Buenrostro-2013bc %}), this paper introduces single-cell ATAC-seq (scATAC-seq). This implementation uses a programmable microfluidic device (Fluidigm C1) to assay hundreds of single cells. The authors explore the cell-to-cell variability of chromatin accessibility and identify _trans_-factors and _cis_-elements involved in this process. The biggest biological claim is that:

> The pattern of accessibility variation in _cis_ across the genome recapitulates chromosome compartments _de novo_, linking single-cell accessibility variation to three-dimensional genome organization.

The method works as follows (Figure 1a and text):

1. Integrated fluidics circuit (IFC) cell capture.
1. Transposition and PCR on the IFC.
2. Libraries collected and PCR amplified with cell-identifying barcodes.
3. Pool the barcoded libraries and sequence.
4. ...
5. Profit?

To demonstrate scATAC-seq, the authors profile 254 individual GM12878 (a HapMap sample) lymphoblastoid cells. They also used this cell line (albeit in bulk) in their [earlier work]({% post_url 2015-07-28-Buenrostro-2013bc %}). Before looking for single-cell variability, they demonstrate that aggregated scATAC-seq data does a good job of recapitulating observations made using bulk ATAC-seq and DNase-seq (where there are 10,000-10,000,000 cells); this recapitulation even extends to the fragment length distributions (which can be used to infer the distribution of nucleosomes). They also profile a host of other cell lines, many from the ENCODE project with extensive genomic profiling data.

In a diploid cell we expect 0, 1, or 2 copies of each regulatory element; __I don't understand how, as the authors state, that Extended Data Figure 3a supports this claim__. Consequently, the data from scATAC-seq are sparse, certainly much more so than scRNA-seq. Other single-cell techniques will also produce sparse data, e.g., single-cell methylation data. Interestingly, the data shown in Figure 1b show that many more cells have a count of 1 than 2; does this mean widespread allele-specfic variability or and undersampling due to low sequencing coverage? Figure 1d shows that you still get sequencing reads from an empty chamber (albeit an order of magnitude fewer than from a non-empty cell)[^empty]!

[^empty]: Supplementary material notes that, "We also find empty chambers contain approximately 100-1,000 fragments which we believe to be from cell free DNA in culture."

The authors state that the sparse nature of the data "makes analysis of cellular variation at individual regulatory elements impractical"; I am somewhat more optimistic.

The main analysis attempts to quantify how different each single-cell's data looks from the aggregated data across a set of genomic features. Following read mapping, peaks were called using MACS2 with the __aggregated data__. From the Supplementary Material, "The top 50,000 non-overlapping 500 bp summits, which we refer to as accessibility peaks, were used for all downstream analysis", but there is no justification given for these cutoffs.

An observed-to-expected measure (a "deviation") was computed for each sample; this attempts to quantify whether the single-cell data are different from another. These deviations are the basis for much of the single-cell level analyses.

The interpretation of peak size is complicated; from the Supplementary Material:

> an element with a longer nucleosome free region (NFR) might yield a greater number of fragments per cell than an element with a shorter NFR that is open in a larger fraction of cells. We therefore would interpret ensemble measures of accessibility as a combination of i) the fraction of cells containing the element and ii) the probability of observing a transposase insertion at that element, defined by the degree of openness if normalized for technical biases.

There is some attempt to correct for technical and sampling error in single-cell data. I found the details in the Supplementary Material a little hard to understand (but I also didn't make a great effort).

By computing correlations of these 'deviations' along the genome, they identified higher-order chromatin structure _a la_ the results observed by chromosome-conformation-capture assays such as Hi-C. The resulting heatmap (Extended Data Figure 9a) looks interesting, but the maximum absolute value of the correlation is 0.2, which seems rather small.

Some other analyses use principal components:

> Principal component (PC) analysis of single-cell deviations across all trans-factors show seven significant PCs, with PC 5 describing changes in DNA abundance throughout the cell cycle. This analysis suggests that high-variance trans-factors are variable independent of the cell cycle (Fig. 2f and Extended Data Fig. 6eâ€“g). The remaining PCs show contributions from several transcription factors, suggesting that variance across sets of trans-factors represent distinct regulatory states in individual cells.

I'm tipping that some of these PCs reflect batch effects rather than any biology.

A neat thing they did was to inhibit potential modifiers of cell-to-cell variability of chromatin accessibility and then ran scATAC-seq. Apart from demonstrating the feasibility of this technique, it also allowed them to conclude that the variability is not just due to differences in cell cycle.

Additional challenges and possible solutions (also from Supplementary Material):

- Some peaks were found to be due to sequencing similarity with mitochondrial DNA and were discarded.
- "We found single-cells varied significantly in the total reads sequenced, stemming from both sequencing coverage and from inherent variation in library diversity... Notably, scATAC-seq data is sparse, therefore we can easily identify and remove all PCR duplicates originating from individual cells without having to use unique molecular identifiers (UMIs), removing a common source of bias in single-cell methods."
- "We observed considerable heterogeneity in the fraction of reads within open chromatin (Figure 1d)"
- "We found cells varied considerably in GC/Tn5 bias. We note that the Tn5 sequence logo is GC rich and is therefore highly correlated with GC bias, thus normalizing for either Tn5 or GC bias yields nearly identical results"
- The analysis is sensitive to the quality of peak annotations.

## Comments and questions on figures

- Figure 1d: Chambers with two cells are indistinguishable (based on library size & fragments in peaks) from those containing one cell. Chambers with dead cells are generally identifiable but not always.
- Extended Data Figure 1f: 0.5-0.75 of reads from each cell are duplicates. This seems high. Also, the fraction of duplicate reads seems to be independent of sequencing coverage; what does this mean?
- Extended Figure 2a: The correlation is computed using only those reads within peaks called from the aggregated scATAC-seq data; how many peaks are uniquely called from aggregated scATAC-seq, bulk ATAC-seq, and DNase-seq data?
- Extended Figure 2j-l: The phase contrast and epifluorescence images of cell #33 look very similar to that of cell #4, yet the ATAC-seq data is rubbish in comparison; why?
  - Supplementary Material suggests that they don't know, which was why they developed filters based on estimated library size and proportion of reads mapping to peaks in order to define 'good' libraries.
- Extended Data Figure 4b-c: Does this suggest that Tn5 bias and GC bias are related?
  - Yes; from Supplementary Material, "Tn5 bias is also highly correlated with GC bias"

## Miscellaneous comments, questions, and thoughts

- "We used either 50x8x8x50 cycle reads from a HiSeq or 76x8x8x76 cycle reads on a NextSeq."
  - What does the "8" refer to in each case? ([Same question]({% post_url 2015-07-28-Buenrostro-2013bc %}))

## Further reading

- [Genome Biology commentary](http://www.genomebiology.com/2015/16/1/172)
